test/uk/gov/hmrc/tai/connectors/TaxCodeChangeConnectorSpec.scala

```diff
@@ -72,8 +61,8 @@ class TaxCodeChangeConnectorSpec extends PlaySpec with MockitoSugar with FakeTai
             "previous" -> Json.arr(
               Json.obj(
                 "taxCode"          -> "code",
-                "startDate"        -> startDate,
-                "endDate"          -> startDate.plusDays(1),
+                "startDate"        -> startDate.toString(),
```

Comment:
Just checking, why are the toString's required now?

Reply:
Json object now requires a JsValueWrapper which a LocalDate cannot satisfy